Python/Tests/Core/AssemblySetup.cs

```diff
@@ -0,0 +1,26 @@
+// Python Tools for Visual Studio
+// Copyright(c) Microsoft Corporation
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the License); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY
+// IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+//
+// See the Apache Version 2.0 License for specific language governing
+// permissions and limitations under the License.
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using TestUtilities;
+
+namespace PythonToolsTests {
+    [TestClass]
+    public sealed class AssemblySetup {
+        [AssemblyInitialize]
+        public static void Initialize(TestContext testContext) => MSTestEnvironment.Initialize();
```

Comment:
We'll also need something similar in the test runner interop module for the UI tests.

Reply:
I don't get it. `TestUtilities.UI` references `TestUtilities` directly, can't we just call `MSTestEnvironment.Initialize()` in the `HostedPythonToolsTestRunner` constructor?

Reply:
Yes. I don't see that change in this PR.

Reply:
But it needs to be called on the VS side, not the MSTest side.

Reply:
Ok